### PR TITLE
Merge release Isetta

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
# PR Details
This PR is to merge release branch isetta to master for release 4.3.0
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docker/checkout.bash [hunk 1]
```diff
@@ -63,3 +66,19 @@ cd ibeo_msgs
...
+
+#rosbridge_suite is a ROS meta-package including all the rosbridge packages.
+git clone https://github.com/usdot-fhwa-stol/rosbridge_suite -b ros2
+
+
+cd ${dir}/src
+
+# git clone --branch master --depth 1 https://github.com/nitroshare/qhttpengine.git
+git clone -b master --depth 1 https://github.com/etherealjoy/qhttpengine.git
+git clone -b isetta --depth 1 https://github.com/usdot-fhwa-OPS/V2X-Hub.git
```
Comment (codygarver) [inline docker/checkout.bash:80]: Need to update to clone final V2X-Hub tag

**Review by codygarver — CHANGES_REQUESTED** [review]
checkout.bash needs to clone correct V2X-Hub branch

File: docker/checkout.bash (hunk 1 above)
Comment (maefromm) [reply docker/checkout.bash:80]: @JonSmet whey do we have a git clone for V2XHub.git here? How does this relate to platform code?

Comment (JonSmet) [reply docker/checkout.bash:80]: It looks like this was added in December 2022 as part of PR #1981 in support of the CARMA Cloud Client (IHP2) development. Specifically, it enabled CARMA Platform to include V2XHub's ccserver package in its build.

Comment (maefromm) [reply docker/checkout.bash:80]: @saina-ramyar  please re-confirm this is needed and which branch it should be pointing to. I assume master? . @SaikrishnaBairamoni once confirmed, I think you can update this file for @codygarver to merge.

Comment (codygarver) [reply docker/checkout.bash:80]: it will need to be changed to whatever tag we end up using in V2X-Hub, `something-something-7.4.0` presumably

Comment (saina-ramyar) [reply docker/checkout.bash:80]: Yes, it is needed for the cellular communication. it should be the main branch, so "develop"

**Pull request merged**